api/v1/helper.go

```diff
@@ -521,7 +521,7 @@ func (cr *SriovIBNetwork) RenderNetAttDef() (*uns.Unstructured, error) {
 		return nil, err
 	}
 	for _, obj := range objs {
-		raw, _ := json.Marshal(obj)
+		raw, _ := json.MarshalIndent(obj, "", "  ")
```

Comment:
just wondering if this will make reading logs better or worse when they are scraped and sent over to a centralized logging and viewed from a web console e.g graylog / ELK stack or similar.

Reply:
hmmm, I hadn't thought of that. I know SumoLogic has auto-formatting capabilities for JSON. So the pretty-printing would be idempotent. Not sure of other tools

Reply:
Just to be clear, this change is so that we can view the plugins config well-formatted in the NetAttachDef CR